cri/v1alpha2/cri_utils.go

```diff
@@ -473,10 +473,10 @@ func (c *CriManager) setupPodNetwork(ctx context.Context, id string, config *run
 		PortMappings: toCNIPortMappings(config.GetPortMappings()),
 	})
 	if err != nil {
-		return "", err
+		return err
```

Comment:
It seems like that there is no need to check error single. A recommended format:

```go
if err := err = c.CniMgr.SetUpPodNetwork(&ocicni.PodNetwork{
        PortMappings: toCNIPortMappings(config.GetPortMappings()),
}); err != nil {
      return err
}
```